easybuild/easyconfigs/b/Biggus/Biggus-0.13.0-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'Biggus'
+version = '0.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/SciTools/biggus/'
+description = """Virtual large arrays and lazy evaluation."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/SciTools/Biggus/archive/']
```

Comment:
If there's a good reason to, then sure, keep it. If not, use PyPI.

Why would they have a release on PyPI if you shouldn't use it to install? That seems weird to me...